apps/ucp-ui/cypress/e2e/institutions.cy.ts

```diff
@@ -102,6 +104,52 @@ describe("institutions", () => {
       });
   });
 
+  it("renders institutions, sorts, checks order, sort again and checks order a second time", () => {
+    cy.loginWithoutWidgetRole();
+
+    const rowRegex = new RegExp(INSTITUTIONS_ROW_TEST_ID);
+
+    cy.waitForLoad();
+
+    cy.findAllByTestId(rowRegex)
+      .eq(0)
+      .invoke("attr", "data-testid")
+      .then((testIdBeforeSort) => {
+        cy.get(INSTITUTIONS_TABLE_SORT_PARENT_CLASS)
+          .eq(0)
+          .find(INSTITUTIONS_TABLE_SORT_ARROW_CLASS_DOWN)
+          .eq(0)
+          .click();
+
+        cy.findAllByTestId(rowRegex);
+
+        cy.waitForLoad();
+
+        cy.findAllByTestId(rowRegex)
+          .eq(0)
+          .invoke("attr", "data-testid")
+          .then((testIdAfterSort) => {
+            expect(testIdAfterSort).not.to.eq(testIdBeforeSort);
+
+            cy.get(INSTITUTIONS_TABLE_SORT_PARENT_CLASS)
+              .eq(0)
+              .find(INSTITUTIONS_TABLE_SORT_ARROW_CLASS_DOWN)
+              .eq(0)
+              .click();
+
+            cy.waitForLoad();
+
+            cy.findAllByTestId(rowRegex)
+              .eq(0)
+              .invoke("attr", "data-testid")
+              .then((testIdFinalSort) => {
+                cy.waitForLoad();
```

Comment:
Why do you need to waitForLoad here?

Reply:
Again, mainly my inexperience with this. I copied this from another test.